src/components/cafes/[id]/Footer/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { FooterBox, updateText } from './Footer.css';
+
+const Footer = () => {
+  const date = new Date();
+  const updatedDate = `${date.getFullYear()}.${String(date.getMonth() + 1).padStart(
+    2,
+    '0'
+  )}.${String(date.getDate()).padStart(2, '0')}`;
+
```

Comment:
이런 format 함수는 별도의 유틸 함수로 분리해도 좋을 것 같아요!

```suggestion
const Footer = (date = new Date()) => {
  const updatedDate = formatDate(date)

```